config/elasticsearch.yml

```diff
@@ -307,7 +307,7 @@ indices.memory.index_buffer_size: 20%
 # operational within the cluster. Its recommended to set it to a higher value
 # than 1 when running more than 2 nodes in the cluster.
 #
-# discovery.zen.minimum_master_nodes: 1
+discovery.zen.minimum_master_nodes: @@MINIMUM_NODES
```

Comment:
This config file is shared with the other cloudformation templates and they would need to be modified to support this config option before merging. Alternatively, can `ElkMinNodes ` be derived from `ElkCapacity` given that it "should be n/2 + 1"?